app/helpers/analyzer.py

```diff
@@ -95,6 +95,8 @@ def _extract_model_settings(self):
             model_settings["should_notify"] = False
 
         model_settings["use_derived_fields"] = self.config_section.getboolean("use_derived_fields")
```

Comment:
There's a fallback option to set default so you could do this in one line:
self.config_section.getboolean("use_derived_fields", fallback=False)